app/src/main/res/values-sw600dp-land/integers.xml

```diff
@@ -7,4 +7,6 @@
     <integer name="phrases_rows">3</integer>
     <integer name="edit_phrases_columns">2</integer>
     <integer name="edit_phrases_rows">3</integer>
+    <integer name="edit_keyboard_columns">10</integer>
```

Comment:
You should be able to omit these from this bucket since they should inherit from the `values-sw600dp` bucket (and both the sw600dp and sw600dp-land look identical)